src/crypto/Recrypt.ts

```diff
@@ -239,3 +240,29 @@ export function generateDeviceAddSignature(jwtToken: string, userMasterKeyPair:
         };
     });
 }
+
+/**
+ * Augments the provided user private key and return both the augmented private key and the augmentation factor.
+ */
+const rotateUsersPrivateKey = (userPrivateKey: Buffer): Future<Error, {newPrivateKey: Buffer; augmentationFactor: Buffer}> => {
```

Comment:
Should we export this too so they could use it without retry if desired?

Reply:
I can't imagine a situation where you would never want to retry.

Reply:
Ya, this method isn't exposed out to callers, it's all wrapped behind the key rotation call. Given the low likelihood of it actually failing I don't think it matters.